packages/studio-ui/src/components/common/TooltipIcon.tsx

```diff
@@ -3,6 +3,10 @@ import { Tooltip } from "react-tooltip";
 import { v4 } from "uuid";
 import { ReactComponent as Info } from "../../icons/info.svg";
 
+const tooltipCssStyle = {
+  maxWidth: "23.5%",
```

Comment:
This width was chosen so that the tooltip would fit nicely within the 25% width sidebar.